### ISSUE TITLE
Columns wrap on mobile

### ISSUE THREAD

**Issue by ultramizer** [dudaster/ele-custom-skin#614]
I have an issue with my columns inside a loop template not wrapping correctly in the posts widget. If i have two or three columns as a row for desktop and tablet, and as a column on mobile, by setting width of each column to 100%, it still shows them as a row in the posts widget on the main site. In Elementor it shows the loop template correctly when i check device sizes, but on the main site where i put the posts widget it doesn't show correctly in Elementor and it also doesn't show in the frontend outside Elementor. The columns just don't wrap but are still side by side in a row. How can i fix that?
And sorry in advance if this has been discussed, i have searched in the issues but couldn't find anything directly related to my question.
Thanks!

**Comment by umm-wdc** [comment]
@dudaster , facing same issue +1

**Comment by umm-wdc** [comment]
@ultramizer , did you get any fix for this ?

**Comment by ultramizer** [comment]
@umm-wdc Nope, unfortunately not. Still waiting for a fix, there also doesn't seem to be a workaround. Sometimes after opening and saving the template for 20+ times it works once, and then stops working again. But i haven't figured out why that is.

**Comment by umm-wdc** [comment]
@ultramizer , the issue seems to be custom column widths, when using the default structure setting it works fine.

**Comment by ultramizer** [comment]
@umm-wdc Can you elaborate on that? I have tried not setting column widths, i have tried setting them in the regular percentage field on the column settings and i have also tried forcing the width using OoohBoi Steroids for Elementor custom Column width where you have additional possibilities to set column width. Nothing works...

**Comment by umm-wdc** [comment]
@ultramizer , i just used the default column setting under Layout tab in sections, and this seems to work without any problem for mobile. 

![image](https://user-images.githubusercontent.com/17593831/143868577-8b21ad9c-86b1-430a-9fc1-633933336c8a.png)

**Comment by ultramizer** [comment]
Alright, but then how would you style the different widths? If want something different than in this structure?

**Comment by umm-wdc** [comment]
> Alright, but then how would you style the different widths? If want something different than in this structure?

For custom widths, we will have to wait for @dudaster  to fix this and release an update

**Comment by ultramizer** [comment]
It would be nice if the bug could be aknowledged by @dudaster in a reasonable timeframe, since im a paid customer and i feel like this is not really a niche feature. Thanks alot!

**Comment by ultramizer** [comment]
@dudaster Any news here? It's impossible to design anything that works on mobile right now with this issue... Thanks for your help!

**Comment by ultramizer** [comment]
@dudaster Any updates on this?

**Comment by ultramizer** [comment]
@dudaster Can you please aknowledge this bug and can we please have a timeframe when this will be fixed?

**Comment by ultramizer** [comment]
@dudaster Any Updates on this? Still not fixed, makes it impossible to work under some circumstances.
